CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,49 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to 
+making participation in our project and our community a harassment-free experience for everyone, 
+regardless of age, body size, disability, ethnicity, gender identity and expression, technology choices and 
+level of experience, nationality, personal appearance, race, religion (or lack thereof), or sexual identity and orientation.
```

Comment:
These are all ordered alphabetically, except for "technology choices and level or experience".

Reply:
And why is that written with an "and"? Are you trying to say "level of experience" applying to "technology"?